File: Java_ChildrenCompetitionSockets/CompetitionModel/src/main/java/competition/model/Participant.java
package competition.model;

public class Participant extends Entity<Integer> {
    private final String username;
    private final String name;
    private final int age;

    public Participant(String username, String name, int age) {
        this.username = username;
        this.name = name;
        this.age = age;
    }

    public String getUsername() {
        return username;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }
}

File: CSharp_ChildrenCompetitionSockets/CSharp_ChildrenCompetitionSockets/model/Participant.cs
namespace CSharp_ChildrenCompetitionGUI.model
{
    public class Participant : Entity<int>
    {
        public string username { get; }
        public string name { get; }
        public int age { get; }

        public Participant(string username, string name, int age)
        {
            this.username = username;
            this.name = name;
            this.age = age;
        }
    }
}